src/wasm.rs

```diff
@@ -51,3 +55,39 @@ impl Encoder {
             .collect()
     }
 }
+
+#[wasm_bindgen]
+pub struct EncodingPacket {
```

Comment:
What is this struct for? It doesn't seem to be used anywhere

Reply:
This data structure incapsulates the raptor frame (packet) that has to be decoded. It contains helpful information and may be used by the client application.
I suppose it's a good idea to have it here because not all JS app devs would like to work with byte-level logic. And this structure hide it from developers and developers free to use this structure without deep knowledge of which  byte is in charge of what.

Reply:
I see. Do you have an example of what the client would need the source block number & encoding symbol id for? I can't think of an obvious use case, since both are pretty internal to the raptorq algorithm

Reply:
Sure
```
let inProgress = {total: -1, collected: new Set() step: None};
<somewhere here the list of frames are being read from some source one by one>
inProgresss.total = <set total number of raptorq frames>;
inProgress.step = Reading;
let decodedPacket = EncodingPacket.deserialize(<uin8arry object with payload>);
let blockNumber = decodedPacket.encoding_symbol_id();
if (inProgress.step == Reading) {
   inProgress.collected.add(blockNumber); //collect the number of processed frames
   let totalProgress = inProgress.collected.size() * 100/inProgress.total; //the percent of processed frames. Used for the UI.
}
```

Reply:
I see. Ok, that use case makes sense to me. I'd like the API to match the Rust library though, and left a comment about that below